testing/pom.xml

```diff
@@ -391,7 +391,7 @@
                 </goals>
                 <configuration>
                   <systemPropertyVariables>
-                    <ccm.version>4.0-beta4</ccm.version>
+                    <ccm.version>trunk</ccm.version>
```

Comment:
Shouldn't this still match what's in the persistent module?

Reply:
I think that idea that Pushkala had was to test against the latest commits in the Cassandra repo in order to catch things early.. But I do agree, we could go with the driver latest as well.. 

The build does fail with the setup I provided.

Reply:
OK so @dougwettlaufer was correct, versions must be the same:

```
Step #4 - "cassandra-4.0": Caused by: java.lang.IllegalArgumentException: Invalid version number: trunk
Step #4 - "cassandra-4.0": 	at com.datastax.oss.driver.api.core.Version.parse(Version.java:95)
Step #4 - "cassandra-4.0": 	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.<clinit>(CcmBridge.java:73)
```

As there is no version for `trunk`, let's go with `rc-2` for now.